docs/vector_store.ipynb

```diff
@@ -481,6 +481,34 @@
     "print(docs)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Multi-modal Vector Store"
+   ]
```

Comment:
Please add more context like "You can also add local, Google Cloud Storage, or web images using the `add_images` method and search by images using the `similarity_search_image` method. "

We should also note that text based searches will return images if images are stored in the vector store